examples/js/vr/GearVRController.js

```diff
@@ -0,0 +1,145 @@
+
+
+THREE.GearVRController = function () {
+
+	THREE.Object3D.call( this );
+
+	var scope = this;
+	var gamepad;
+
+	var axes = [ 0, 0 ];
+	var touchpadIsPressed = false;
+	var triggerIsPressed = false;
+	var angularVelocity = new THREE.Vector3();
+
+	this.matrixAutoUpdate = true;
+
+	function findGamepad() {
+
+		var gamepads = navigator.getGamepads && navigator.getGamepads();
+
+		for ( var i = 0; i < 4; i ++ ) {
+
+			var gamepad = gamepads[ i ];
+
+			if ( gamepad && ( gamepad.id === 'Gear VR Controller' ) ) {
+
+				return gamepad;
+
+			}
+
+		}
+
+	}
+
+	this.setSkyBox = function ( skyBox ) {
+
+		if ( 'SamsungChangeSky' in window ) {
+
+			window.SamsungChangeSky( skyBox );
```

Comment:
I'm not sure this should be a feature of the `GearVRController`?

Reply:
So remove it and leave it up for the developers?

Reply:
I think so, yeah.
We also show be careful of not adding to many images to the repo. It's already pretty big.

Reply:
Also... Is `SamsungSky` more performant than using `scene.background`?

Reply:
The SamsungSky function will change the skybox before you enter webvr. Not inside Three.js
See http://developer.samsung.com/internet/gearvr/web-guide for this

Reply:
As for the images i'll see if i can downsize them

Reply:
Why not using one of the skyboxes textures already available?
https://github.com/mrdoob/three.js/tree/dev/examples/textures/cube

Reply:
Sorry wasn't aware of that. I'll edit it.